uiowa/uiowa
# [ITS] Filter focus fix #3

Resolves https://github.com/uiowa/uiowa/issues/7447

# How to test

```
ddev composer install && ddev blt ds --site=its.uiowa.edu && ddev drush @its.local uli /alerts/status-history
```

1. Check to see if the form works okay after autosubmit is turned off.
